modules/addons/supportpin/hooks.php

```diff
@@ -45,7 +45,7 @@
         $supportPinMenu = $primarySidebar->addChild('supportPinMenu',
             array(
                 'name' => 'SupportPin',
-                'label' => 'Support Pin',
+                'label' => $admin_table_pin,

```

Comment:
This is not possible, for this `$_lang = $vars['_lang'];` must be initalized and `$_lang['admin_table_pin']` must be used to access it
```suggestion
                'label' => $_lang['admin_table_pin'],
```